sonaranalyzer-dotnet/src/SonarAnalyzer.Utilities/Rules.Description/S2201.html

```diff
@@ -7,6 +7,8 @@
   <li> any method on <code>string</code>, <code>int</code>, ..., <code>System.Collections.Immutable.ImmutableArray&lt;T&gt;</code>,
   <code>ImmutableHashSet&lt;T&gt;</code>, ... </li>
 </ul>
+<p>Note: although <code>string.Intern</code> method has a side effect, ignoring its return value is still suspicious as it is the only reference
```

Comment:
I believe you should either use "although **the** `string.Intern` method has a side effect" OR "although `string.Intern` has a side effect" (preferably the 2nd as it has less words)

Reply:
I actually had the same reflection when modifying RSPEC, and ended up confusing myself 😅.
Updated RSPEC to 2nd solution